tests/istio.mk

```diff
@@ -66,6 +66,11 @@ DEFAULT_EXTRA_E2E_ARGS += --galley_hub=${HUB}
 EXTRA_E2E_ARGS ?= ${DEFAULT_EXTRA_E2E_ARGS}
 
 e2e_simple: istioctl generate_yaml e2e_simple_run
+e2e_simple_cni: istioctl
+e2e_simple_cni: export ENABLE_ISTIO_CNI=true
+e2e_simple_cni: export EXTRA_HELM_SETTINGS=--set istio-cni.excludeNamespaces={} --set istio-cni.pullPolicy=IfNotPresent --set istio-cni.tag=$(ISTIO_CNI_DOCKER_TAG) --set istio-cni.hub=$(ISTIO_CNI_DOCKER_HUB)
```

Comment:
the helm implementation could just reuse the global hub here.

Reply:
@sdake - I don't want the code to assume that the Istio/istio and istio/cni hub and tag are the same.  In most cases I think people will have one be a known good hub/tag and the other being the hub/tag for the rep they are testing